README.md

```diff
@@ -184,6 +184,7 @@ Contributions of any kind welcome, just follow the [guidelines](.github/CONTRIBU
 - [confetti](https://github.com/ArthurData/quarto-confetti) - This extension uses [canvas-confetti](https://github.com/catdad/canvas-confetti) a JavaScript library to send some confetti in your `revealjs` presentation.
 - [code-insertion](https://github.com/feynlee/code-insertion) - This extension enables code insertion immediately before and/or after a post/page for website and blog projects.
 - [snow](https://github.com/EmilHvitfeldt/quarto-snow) - This extension adds falling snowflakes in `html` and `revealjs` formats.
+- [swissbiopics](https://github.com/zachcp/quarto-swissbiopics) - This extension adds shortcode to embed [SwissBioPics](https://www.swissbiopics.org/) library of interactive biological images in `html` and `revealjs` formats.
```

Comment:
Only one sentence for description.
Proper case for `SwissBioPics`.

Also, the extension is only valid for HTML based formats

Reply:
I made some changes in 2cddcf87e83365d9730be678d92004db3721ab97